spec/models/spree/country_spec.rb

```diff
@@ -1,9 +1,15 @@
+# frozen_string_literal: true
+
 require 'spec_helper'
 
 describe Spree::Country do
   it "can find all countries group by states required" do
-    country_states_required= Spree::Country.create({:name => "Canada", :iso_name => "CAN", :states_required => true})
-    country_states_not_required= Spree::Country.create({:name => "France", :iso_name => "FR", :states_required => false})
+    country_states_required = Spree::Country.create({ name: "Canada",
+                                                      iso_name: "CAN",
+                                                      states_required: true })
+    country_states_not_required = Spree::Country.create({ name: "France",
+                                                          iso_name: "FR",
+                                                          states_required: false })
```

Comment:
curly braces are actually not required, right?

Reply:
this was testing dead code, it's removed.